CSharpSourceCode/Battle/Grenades/GrenadesMissionLogic.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System.Linq;
+using TaleWorlds.Core;
+using TaleWorlds.Engine;
+using TaleWorlds.InputSystem;
+using TaleWorlds.Library;
+using TaleWorlds.MountAndBlade;
+using TaleWorlds.ObjectSystem;
+using TOW_Core.Utilities;
+
+namespace TOW_Core.Battle.Grenades
+{
+    public class GrenadesMissionLogic : MissionLogic
+    {
+        public GrenadesMissionLogic()
+        {
+        }
+
+        public override void OnMissionTick(float dt)
+        {
+            base.OnMissionTick(dt);
+            if (Input.IsKeyPressed(InputKey.L))
```

Comment:
Should be removed before final merge.

Reply:
Sure. I want to move this to DebugMissionLogic i am working on.